### PR TITLE
Documented SharedAccessSignature and BlobEndpoint in Azure Storage

### PR DESCRIPTION
**Pull request by MarcoBedulli**

### CODE REVIEW

**Comment by MarkvanMents** [comment]
@MarcoBedulli Are these changes in the current runtime/deployment or is this PR for the future?

**Comment by MarcoBedulli** [comment]
Starting from Mendix 8.1, there will be this future.

**Comment by MarkvanMents** [comment]
Thanks, Marco. I'll label if for future release.

File: content/refguide/custom-settings.md [hunk 1]
```diff
@@ -150,6 +150,8 @@ These settings can be used to store files using the Microsoft Azure blob storage
 | com.mendix.core.StorageService | Has to be set to 'com.mendix.storage.azure' to select Azure as the storage service |   |
 | com.mendix.storage.azure.AccountName | Account name to authenticate with the azure blob storage service |   |
 | com.mendix.storage.azure.AccountKey | Account key to authenticate with the azure blob storage service |   |
+| com.mendix.storage.azure.SharedAccessSignature | Provides delegated access to resources in your storage account. [Shared Access Signature](https://docs.microsoft.com/en-us/azure/storage/common/storage-dotnet-shared-access-signature-part-1)|   |
+| com.mendix.storage.azure.BlobEndpoint |Set the blob endpoint. This setting is required when authenticate by SharedAccessSignature is used|   |
```
Comment (MarkvanMents) [inline content/refguide/custom-settings.md]: image

@MarcoBedulli - it is not clear to me what values should be provided for these. Is it the **SAS Token** and the **Storage Resource URI** described in https://docs.microsoft.com/en-us/azure/storage/common/storage-sas-overview or is it something else?

Comment (MarkvanMents) [reply content/refguide/custom-settings.md]: These are provided during the process and have the same names as the settings. Therefore no need to explain further.

**Pull request merged**